code/datums/keybinding/admin.dm

```diff
@@ -10,8 +10,12 @@
 	name = "admin_say"
 	full_name = "Admin say"
 	description = "Talk with other admins."
+	keybind_signal = COMSIG_KB_ADMIN_ASAY_DOWN
 
 /datum/keybinding/admin/admin_say/down(client/user)
+	. = ..()
+	if(.)
+		return
```

Comment:
Just use `if(..())`

Reply:
Don't use `if(..())`, preserve the returns.
Can you revert these changes?
If you do this and redefine the proc one more level the return value won't be passed to it.